codalab/model/bundle_model.py

```diff
@@ -774,7 +774,7 @@ def do_update(connection):
     def get_bundle_state(self, uuid):
         result_dict = self.get_bundle_states([uuid])
         if uuid not in result_dict:
-            raise UsageError('Could not find bundle with uuid %s' % uuid)
+            raise NotFoundError('Could not find bundle with uuid %s' % uuid)
```

Comment:
I took a closer look, and I realized that NotFoundError is actually a
subclass of UsageError (and I also saw Stephen's note) so what I said was
incorrect - would you mind going back to NotFoundError?  Sorry about the
confusion.

LGTM otherwise

On Thu, Aug 11, 2016 at 11:13 AM, Karan Desai <notifications@github.com>
wrote:

> In codalab/model/bundle_model.py
> <https://github.com/codalab/codalab-cli/pull/566#discussion_r74389929>:
>
> > @@ -774,7 +774,7 @@ def do_update(connection):
> >      def get_bundle_state(self, uuid):
> >          result_dict = self.get_bundle_states([uuid])
> >          if uuid not in result_dict:
> > -            raise UsageError('Could not find bundle with uuid %s' % uuid)
> > +            raise NotFoundError('Could not find bundle with uuid %s' % uuid)
>
> @percyliang <https://github.com/percyliang>:
> Very well, I reverted back the changes. ✅
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/codalab/codalab-cli/pull/566/files/c11e88b533679bb1639f2c18421f81b71d33661b#r74389929>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAakuO4ufsAFVu_QOyUNhVuBtMb8rIP7ks5qeuetgaJpZM4JgDcj>
> .
>